src/test/java/testtypes/GetTestTypesByIdData.java

```diff
@@ -3973,7 +3973,7 @@ public void validateTestTypeValidVehicleConfigurationIncompatibleFilters() {
         testTypeSteps.validateRawData("\"No resources match the search criteria.\"");
     }
 
-
+    @Ignore("No longer a valid test case")
```

Comment:
For the the tests you added @Ignore tag please also remove @Test tag so they won't appear in the test report anymore.